### ISSUE TITLE
Rubocop 0.47.1 ignores its configuration

### ISSUE THREAD

**Issue by PascalTurbo** [bbatsov/rubocop#3968]
With the upgrade from rubocop 0.46 to 0.47.1 rubocops seems to ignore the following tags

```ruby    
# rubocop:disable Style/DocumentationMethod Style/Documentation
```

and it seems to ignore the configuration, too.

For reproducing simply checkout https://github.com/feedjira/feedjira and run it once against 0.46 and then against 0.47.1. With the newest version it starts to check our rspec files and ignores the disable tags.

**Comment by halo** [comment]
I experience something similar since 0.47.1. After upgrading, the `Metrics/BlockLength` cop started complaining about my `*_spec.rb` files, saying that the "Block has too many lines". Which, as far as I remember, should not be a concern for spec files. With 0.46.0, there are no complaints by any cop.

**Comment by mikegee** [comment]
I suspect https://github.com/bbatsov/rubocop/commit/157b947473e32e6e79bedfa57798fdf9fc6a0650 is the problem with `Metrics/BlockLength` complaining about specs.

**Comment by halo** [comment]
Thank you @mikegee. I was unaware that the default exclude had been removed (or rather "moved" from `config/default.yml` to `.rubocop.yml`). I added that to my own `.rubocop.yml`, which solves the issue for me. 🙆‍♂️

**Comment by jonas054** [comment]
So no bug. Indented behavior as far as I can see.
